Modules/_ssl.c

```diff
@@ -3042,6 +3152,7 @@ _ssl__SSLContext_impl(PyTypeObject *type, int proto_version)
         return NULL;
     }
 
+    // This is the constructor, no need to lock it.
```

Comment:
Technically, it's not even possible to lock it since the lock depends on an attribute that is still not yet initialized.

Reply:
Eh, I don't think it's worth changing. I don't want somebody to freak out in the future because this function is missing locks.

Reply:
My point was more on the fact that you cannot even use the macro since `self->lock` is not even set. Having "no need to lock it" may indicate that it's *possible* to use the macro which is not he case.

Reply:
I guess I'll move the comment.